command/harness/setup.go

```diff
@@ -42,7 +42,7 @@ type SetupVMResponse struct {
 }
 
 var (
-	healthCheckTimeout = 3 * time.Minute
+	healthCheckTimeout = 5 * time.Minute
```

Comment:
@vistaarjuneja increased this because windows builds are taking more than 3 minutes in some cases. had few failures in local testing.